website/content/community/_index.md

```diff
@@ -127,6 +127,19 @@ registries, and so forth. `inv -l` lists the available tasks.
 To build and deploy MetalLB to a local development environment using a kind
 cluster, run `inv dev-env`.
 
+When you're developing, running components at the command line and
+having them attach to a cluster might be more convenient than
+redeploying them to a cluster over and over.
+
+For the controller, the `-kubeconfig` and `-config-ns` command-line flags
+are needed.  Speakers need those and `-node-name`.
+
+For example:
+
+ metallb$ go run ./controller/main.go ./controller/service.go -config-ns metallb-system -kubeconfig $KUBECONFIG
```

Comment:
As a future idea, maybe a tasks.py addition of a "run-controller" command would be a little easier.  same with "run-speaker".